benchmark.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
`chown +x` to make this executable?

Reply:
resolved with https://github.com/apache/flink-benchmarks/pull/90/commits/74e4f4f56dab07c70e03690279f96a8f168830f4